app/models/mdm/web_page.rb

```diff
@@ -1,4 +1,5 @@
 # Web page requested from a {#web_site}.
```

Comment:
This comment should be above the `class` line not above the `require` because the comment now reads as explaining why the `require 'webrick'` is there.

You should have a comment explaining why `require 'webrick'` is necessary.